decred.go

```diff
@@ -144,17 +144,42 @@ func main() {
 
 	amgr.AddAddresses([]net.IP{net.ParseIP(cfg.Seeder)})
 
+	ctx, shutdown := context.WithCancel(context.Background())
+	killSwitch := make(chan os.Signal, 1)
+	signal.Notify(killSwitch, os.Interrupt)
```

Comment:
`os.Interrupt` will catch `^C`, but we need to catch `syscall.SIGTERM` as well, otherwise the graceful shutdown won't be triggered if the process is ended by other means, e.g. with `kill`

Reply:
I suppose it's fair to assume no one is running dcrseeder on Windows, so we don't have to bother with conditional compilation.  Will just add SIGTERM right here instead of making it more complicated unless we think it's worth supporting Windows.